src/components/AcceptDialog.tsx

```diff
@@ -44,9 +44,13 @@ const AcceptDialog: FunctionComponent<Props> = (props) => {
         );
         router.push(props.redirect, undefined, { shallow: true });
       } catch (err) {
-        console.error(err);
+        console.log(err);
```

Comment:
I think we should stick with console.error just because it is easier to spot the error in the console in the browser as it shows up in red

Reply:
I've made the change!